src/components/Theme/style.scss

```diff
@@ -41,6 +41,7 @@
   height: calc(100% - #{$non-content-height});
   position: relative;
   display: flex;
+  flex-direction: column;
```

Comment:
Why does `flex-direction: column;` need to be defined for `.content` now?

Reply:
The element in ScreenLayout are direct children of this class. Therefore we need to make sure that all the children of this class are displayed vertically